polyfill/ci_codecov_test262.sh

```diff
@@ -23,6 +23,7 @@ node runtest262.mjs "test262/test/built-ins/Date/*/toTemporalInstant/*.js" || [
 for subdir in $subdirs; do
   node runtest262.mjs "$subdir/**" || failed=1
 done
+node runtest262.mjs "test262/test/intl402/**/*[tT]emporal*.js" || failed=1
```

Comment:
It occurred to me that we should probably run all the tests in `test262/test/intl402/DateTimeFormat` since the polyfill patches DateTimeFormat. (There's also a `test262/test/intl402/Intl/DateTimeFormat` folder that should be merged with the former. I'll open a test262 issue for that.)

I think that's out of scope for this pull request. Especially since I tried it and there are some failures. Just noting it for the future.

Reply:
Yeah, I had the same thought right after I sent the Test262 PR to fix the four tests that broke in `test262/test/intl402/**/*[tT]emporal*.js`, but it was late and I didn't want to chew off fixing all the others. I'll send a Test262 PR to fix those tests too, assuming it's easy.

Reply:
Also, I noticed that `test262/test/built-ins/Date/*/toTemporalInstant/*.js` doesn't seem to exist. Did it move?  Or are we anticipating a time later when we will populate this folder?

Reply:
The test262 issue I opened is https://github.com/tc39/test262/issues/3763

There just aren't any tests for `Date.prototype.toTemporalInstant` yet. You're correct that the script anticipates them.

Reply:
> It occurred to me that we should probably run all the tests in `test262/test/intl402/DateTimeFormat` since the polyfill patches DateTimeFormat. {snip}
> I think that's out of scope for this pull request. Especially since I tried it and there are some failures. Just noting it for the future.

Yep. There were 37 failures. I filed https://github.com/tc39/proposal-temporal/issues/2471 to track the fixes for them.  Definitely out of scope for this PR to fix all those!

On CLDR 41 and earlier, there were 5 more failures caused by CLDR 42 compatibility. I filed https://github.com/tc39/test262/pull/3765 to fix those. At this point, there shouldn't be any more CLDR42-related compatibility issues remaining in Test262.  I tested the fixes both locally and in CI against the Temporal polyfill to make sure that the CLDR 42 issues weren't masking legit polyfill issues, but all 5 passed. 

But I don't think we need to block merging this PR on those 5 Test262 fixes being merged because those tests aren't actually testing Temporal objects... and because Intl.DTF tests don't pass anyways.